controllers/getArtComments.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { fetchArticleComments } = require('../Models/getArtComments.models')
+const db = require('../db/connection')
+
+
+exports.getArticleComments = (req, res) => 
+{
+    fetchArticleComments(req).then((data) =>
+    {
+        res.status(200).send({ data })
+    }).catch(({ err }) => 
+    {
+        res.status(404).send()
+    })
```

Comment:
You should be sending the error response within your error handling middleware not in the controller as it can create repetitive code. the best thing you can do is have one place that handles all the errors in the server so if some errors are the same then you only need to write code for that response once. Please have a look into the notes for error handling in express